### PR TITLE
Show disabled user count in occ user:report

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @wobben

### CODE REVIEW

File: core/Command/User/Report.php [hunk 1]
```diff
@@ -73,6 +77,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		$rows[] = [' '];
 		$rows[] = ['user directories', $userDirectoryCount];
 
+		$disabledUsers = $this->config->getUsersForUserValue('core', 'enabled', 'false');
+		$disabledUsersCount = count($disabledUsers);
```
Comment (nickvergessen) [inline core/Command/User/Report.php:81]: :see_no_evil: should maybe add a dedicated function for this instead. But I geuss disabled users shouldn't be too many :-X

**Review by nickvergessen — APPROVED** [review]
:shark:

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
@MorrisJobke backport?

**Pull request merged**